docs/internals.rst

```diff
@@ -588,6 +588,30 @@ This split is done to avoid self-triggering and triggering coroutines multiple
 times. See the `Scheduling with timing` section for details on how this is
 used.
 
+``VlDynamicTriggerScheduler``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Like ``VlTriggerScheduler``, it manages processes that await triggers. However,
```

Comment:
```suggestion
Like ``VlTriggerScheduler``, ``VlDynamicTriggerScheduler`` manages processes that await triggers. However,
```

and re-fill paragraph.

(Typically clearer not to assume the headings were just read)